CONTRIBUTING.md

```diff
@@ -1,65 +1,251 @@
+      consider pointing the reviewer to the specific files or lines
+      you'd like the reviewer to check.
+
+1. If your PR isn't ready for review,
+   [set it to Draft][set-draft] and, if possible,
+   [apply the **st.WIP** label][apply-wip-label].
+
+1. Click **Create pull request**. This starts the review process.
+
+### Respond to reviewer feedback
+
+1. Read and address comments from the reviewer.
+   Acknowledge all comments with at least a [reaction emoji][reaction-emoji].
+   If the changes are technically or grammatically correct, make them.
+   Reply to comments that you want to discuss.
+   Remember that all comments are public. Keep your replies civil and relevant.
+
+1. When your PR is ready for another round of feedback,
+   re-request a review or write a comment with `PTAL @[reviewer]`.
+   (PTAL means _please take another look_.)
+
+1. Once your PR is approved, a reviewer [merges it][merges-pr].
+
+## Reviewers
+
+### General guidelines for reviews
+
+* Technical review comes before content, or "copy", review.
+* Lean toward velocity. Aim for _improvement_ over absolute _perfection_.
+* Check the whole page. Sometimes new content contradicts unchanged
+  content elsewhere on the same page.
+* Prefer giving an overall note when there's a repeated problem,
+  rather than commenting on every instance.
+* Try to **approve** with suggestions wherever possible.
+  Use **request changes** only when the change can harm an end user's
+  work.
+* Close PRs that violate [Flutter's Code of Conduct][code-of-conduct].
+
+### Perform a technical review
+
+In general, follow these practices.
+
+* Critique the _technical aspects_ of the PR.
+  If technical content could be made less complex or more detailed,
+  advise the writer that this is the case in your review.
+  Still _approve the PR_ but only if your changes are included.
+* Provide suggested changes with corrected copy to make your changes clear.
+
+To conduct a consistent technical review in the Flutter website:
+
+1. [Review a PR][review-pr] when it meets the following conditions:
+
+   1. The writer [requested your review][request-review] of the PR.
+
+   1. The writer applied the [**review.tech**][review-tech-label]
+      label to the PR, if possible.
+
+   1. The writer didn't set the PR to [Draft mode][set-draft].
+      You can provide feedback to a draft issue,
+      but you should understand that the author's still working on the draft.
+
+1. Validate technical accuracy.
+
+   1. Check any step-by-step procedures and code examples.
+
+   1. If code examples have been added or changed,
+      verify that new code is included in CI and that tests are passing.
+
+   1. Verify that the code remains consistent with the
+      [Flutter Tree Hygiene guide][flutter-tree]
+      and the [Flutter code style guide][flutter-style].
+
+1. When [adding comments][add-comments],
+   state explicitly if you're making a suggestion or
+   pointing out something that needs to change.
```

Comment:
I usually use "nit" or "nitpick" to label a comment as a nice-to-have.

Reply:
I had a whole thing in here about [conventional comments](https://conventionalcomments.org) to set a standard, but it was deemed too complex. You can do as you describe here, of course.